provider/aws/dist/rack.json

```diff
@@ -104,9 +104,11 @@
       "Value": { "Ref": "EncryptionKey" }
     },
     "Gateway": {
+      "Condition": "BlankExistingVpc",
       "Value": { "Ref": "Gateway" }
     },
     "GatewayAttachment": {
+      "Condition": "BlankExistingVpc",
```

Comment:
I was using the same condition here: https://github.com/convox/rack/blob/fda3718f67981724f866149d1e5d293cf8fee130/provider/aws/dist/rack.json#L656